README.md

```diff
@@ -104,8 +98,8 @@ To get a local copy up and running follow these simple example steps.
 
 ## 🔭 Future Features <a name="future-features"></a>
 
-- [] **[Creating Calculator App Using React]**
-- [] **[Implementing better UI]**
+- [] **[new_feature_1]**
+- [] **[new_feature_2]**
```

Comment:
- [OPTIONAL] I think it would be better if you add some specific and understandable future features instead of `new_feature_1,...` 😁 🙃